assets/css/omise-css.css

```diff
@@ -21,3 +21,92 @@ fieldset.card-exists {
 #new_card_info:checked ~ fieldset.card-exists {
     display: block;
 }
+
```

Comment:
Note: would be nice to ignore the CSS file for now as I've provided a unifying-css-files already at the pull request https://github.com/omise/omise-woocommerce/pull/113 (currently there are some stylesheets that are duplicated and bad-organize).